src/lumigo_tracer/spans_container.py

```diff
@@ -180,11 +185,15 @@ def update_event_response(
             self.http_span_ids_to_send.add(update.get("id") or last_event["id"])
 
     def add_exception_event(self, exception: Exception) -> None:
+        """
+        Should be called only in `except` scope.
+        """
         if self.function_span:
             self.function_span["error"] = {
                 "type": exception.__class__.__name__,
                 "message": exception.args[0] if exception.args else None,
                 "stacktrace": traceback.format_exc(),
+                "frames": extract_frames_from_exception() if Configuration.verbose else [],
```

Comment:
Get Exception trace from parm

Reply:
The `exception` doesn't have the stack frames, I need to use `inspect` in order to go over the stack.